bibliopixel/remote/control.py

```diff
@@ -0,0 +1,140 @@
+import multiprocessing
+import inspect
+from .. animation import BaseAnimation
```

Comment:
or `from .. animation import animation` and then use `animation.BaseAnimation` and `animation.State`.